### ISSUE TITLE
Single annotation may be placed on the same line

### ISSUE THREAD

**Issue by vanniktech** [pinterest/ktlint#248]
According to both the Kotlin and Android style guidelines a single annotation may be placed on the same line:

```kotlin
@Test fun allTheThings()
```

I'd like to enforce this in my projects however the rule definition isn't really strict about it. I guess the main intention for everyone is that it is the same across the project. Either on the same line or on a separate one. Maybe that's how ktlint can figure out whether it should be forced or not since ktlint is reluctant to add extra configurations.

**Comment by shashachu** [comment]
This is passing in the current version of `experimental:annotation`.
